signus/freedesktop/io.github.signus_game.signus.metainfo.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="desktop">
+  <!--Created with jdAppStreamEdit 7.0-->
+  <id>io.github.signus_game.signus</id>
+  <name>Signus: The Artefact Wars</name>
+  <summary>turn-based strategy game</summary>
+  <developer_name></developer_name>
```

Comment:
Developer could be either `Valacirca` (the company that made the original commercial version) or `Signus game team`.